schemas/cgTranscripts.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var cgTranscriptSchema = new Schema({
+  bitsId: String,
```

Comment:
Rename this field to `idNumber`

Reply:
Is it really important to change this? I chose `bitsId` so that it is easily understood which ID is being referred to ( idNumber is ambiguous ) . This name convention is being used across the app and the server now, so changing it will take a lot of time. Can I keep it as it is @sohailrajdev97 ?